string iface
string product
string device_type
string hardware_address
string driver

string ETHERNET_DEVICE =Ethernet
string WIFI_DEVICE=802.11 WiFi
string GSM_DEVICE=GSM
string CDMA_DEVICE=CDMA
string MOBILE_BROADBAND_DEVICE=Mobile Broadband
string WIMAX_DEVICE=WiMAX
string OTHER_DEVICE=Other